### PR TITLE
Add jira 1.0.10

### PR DESCRIPTION
**Pull request by ericdill**
Followon from #1015

cc @parente 

@pmlandwehr do you want to be a maintainer on this?

### CODE REVIEW

**Comment by pmlandwehr** [comment]
Sure, you can add me.

**Comment by ericdill** [comment]
@pmlandwehr Added.

File: recipes/jira/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    - setuptools >=20.10.1
+    - defusedxml
+
+test:
+  imports:
+    - {{ name }}
+
+about:
+  home: https://github.com/pycontribs/jira
+  license: BSD 2-Clause
```
Comment (ericdill) [inline recipes/jira/meta.yaml]: Ah good catch. Yeah I'll do that

**Comment by ericdill** [comment]
@ocefpaf Can you review this? I think it's good to go.

**Pull request merged**